Native image generation failure for `integration-tests/keycloak-authorization` in resteasy-reactive

### Describe the bug

I've just noticed it in [this build log](https://pipelines.actions.githubusercontent.com/serviceHosts/db7e007d-ef45-446a-a481-86fd29e0a95c/_apis/pipelines/1/runs/167795/signedlogcontent/73?urlExpires=2022-09-16T08%3A20%3A14.9904555Z&urlSigningMethod=HMACV1&urlSignature=G4jxvtdxv9WQaE%2FH4gIoC1qoHeFV5zqGf%2BO94eVQTn0%3D):
```
2022-09-16T07:59:26.0151477Z [INFO] --- quarkus-maven-plugin:999-SNAPSHOT:build (default) @ quarkus-integration-test-keycloak-authorization ---
2022-09-16T07:59:27.5576125Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyReader 'org.jboss.resteasy.plugins.providers.IIOImageProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.5661192Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.sse.SseEventProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.5695234Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.multipart.MultipartWriter' handles so this Provider will be ignored
2022-09-16T07:59:27.5890494Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyReader 'org.jboss.resteasy.plugins.providers.jackson.ResteasyJackson2Provider' handles so this Provider will be ignored
2022-09-16T07:59:27.5994264Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyReader 'org.jboss.resteasy.plugins.providers.jaxb.JAXBElementProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.5995652Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.multipart.MultipartRelatedWriter' handles so this Provider will be ignored
2022-09-16T07:59:27.6022466Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyReader 'org.jboss.resteasy.plugins.providers.jaxb.JAXBXmlTypeProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6023824Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.multipart.MultipartFormDataWriter' handles so this Provider will be ignored
2022-09-16T07:59:27.6025340Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.multipart.MapMultipartFormDataWriter' handles so this Provider will be ignored
2022-09-16T07:59:27.6026638Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.DefaultBooleanWriter' handles so this Provider will be ignored
2022-09-16T07:59:27.6028048Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.StringTextStar' handles so this Provider will be ignored
2022-09-16T07:59:27.6029140Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.jaxb.MapProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6042773Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.jaxb.CollectionProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6046190Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyReader 'org.jboss.resteasy.plugins.providers.DataSourceProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6057661Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.JaxrsFormProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6143978Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.FileRangeWriter' handles so this Provider will be ignored
2022-09-16T07:59:27.6156284Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyReader 'org.jboss.resteasy.plugins.providers.DefaultTextPlain' handles so this Provider will be ignored
2022-09-16T07:59:27.6175810Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.FormUrlEncodedProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6178500Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.multipart.XopWithMultipartRelatedWriter' handles so this Provider will be ignored
2022-09-16T07:59:27.6257730Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.DefaultNumberWriter' handles so this Provider will be ignored
2022-09-16T07:59:27.6298407Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.SourceProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6299747Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyReader 'org.jboss.resteasy.plugins.providers.multipart.MimeMultipartProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6301488Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyReader 'org.jboss.resteasy.plugins.providers.jaxb.JAXBXmlRootElementProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6302917Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.multipart.MultipartFormAnnotationWriter' handles so this Provider will be ignored
2022-09-16T07:59:27.6317707Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.FileProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6330432Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyReader 'org.jboss.resteasy.plugins.providers.DocumentProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6336434Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.multipart.ListMultipartWriter' handles so this Provider will be ignored
2022-09-16T07:59:27.6348475Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.ByteArrayProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6353626Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyReader 'org.jboss.resteasy.plugins.providers.jaxb.JAXBXmlSeeAlsoProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6365353Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.AsyncStreamingOutputProvider' handles so this Provider will be ignored
2022-09-16T07:59:27.6422613Z [WARNING] [io.quarkus.resteasy.reactive.server] Unable to determine which type MessageBodyWriter 'org.jboss.resteasy.plugins.providers.InputStreamProvider' handles so this Provider will be ignored
2022-09-16T07:59:28.6950944Z [WARNING] [io.quarkus.deployment.steps.CombinedIndexBuildStep] Failed to index int: Class does not exist in ClassLoader QuarkusClassLoader:Deployment Class Loader: PROD@599739be
2022-09-16T07:59:29.9223314Z [INFO] [io.quarkus.deployment.pkg.steps.JarResultBuildStep] Building native image source jar: /home/runner/work/quarkus/quarkus/integration-tests/keycloak-authorization/target/quarkus-integration-test-keycloak-authorization-999-SNAPSHOT-native-image-source-jar/quarkus-integration-test-keycloak-authorization-999-SNAPSHOT-runner.jar
2022-09-16T07:59:30.7083218Z [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Building native image from /home/runner/work/quarkus/quarkus/integration-tests/keycloak-authorization/target/quarkus-integration-test-keycloak-authorization-999-SNAPSHOT-native-image-source-jar/quarkus-integration-test-keycloak-authorization-999-SNAPSHOT-runner.jar
2022-09-16T07:59:30.8613306Z [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildContainerRunner] Using docker to run the native image builder
2022-09-16T07:59:30.9028413Z [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildContainerRunner] Checking image status quay.io/quarkus/ubi-quarkus-native-image:22.2-java17
2022-09-16T07:59:31.1085386Z 22.2-java17: Pulling from quarkus/ubi-quarkus-native-image
2022-09-16T07:59:31.1096603Z Digest: sha256:ee128914b55187c256de677fe862c7c4335be2a48093606b3aa10ef07235f876
2022-09-16T07:59:31.1102402Z Status: Image is up to date for quay.io/quarkus/ubi-quarkus-native-image:22.2-java17
2022-09-16T07:59:31.1107191Z quay.io/quarkus/ubi-quarkus-native-image:22.2-java17
2022-09-16T07:59:31.7147877Z [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Running Quarkus native-image plugin on GraalVM 22.2.0 Java 17 CE (Java Version 17.0.4+8-jvmci-22.2-b06)
2022-09-16T07:59:31.7400910Z [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker run --env LANG=C --rm --user 1001:121 -v /home/runner/work/quarkus/quarkus/integration-tests/keycloak-authorization/target/quarkus-integration-test-keycloak-authorization-999-SNAPSHOT-native-image-source-jar:/project:z --name build-native-uoyVo quay.io/quarkus/ubi-quarkus-native-image:22.2-java17 -J-Dsun.nio.ch.maxUpdateArraySize=100 -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager -J-Dlogging.initial-configurator.min-level=500 -J-Dvertx.logger-delegate-factory-class-name=io.quarkus.vertx.core.runtime.VertxLogDelegateFactory -J-Dvertx.disableDnsResolver=true -J-Dio.netty.leakDetection.level=DISABLED -J-Dio.netty.allocator.maxOrder=3 -J-Duser.language=en -J-Duser.country=US -J-Dfile.encoding=UTF-8 --features=io.quarkus.runner.Feature,io.quarkus.runtime.graal.ResourcesFeature,io.quarkus.runtime.graal.DisableLoggingFeature -J--add-exports=java.security.jgss/sun.security.krb5=ALL-UNNAMED -J--add-opens=java.base/java.text=ALL-UNNAMED -J--add-opens=java.base/java.io=ALL-UNNAMED -J--add-opens=java.base/java.lang.invoke=ALL-UNNAMED -J--add-opens=java.base/java.util=ALL-UNNAMED -H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy\$BySpaceAndTime -H:+AllowFoldMethods -J-Djava.awt.headless=true --no-fallback --link-at-build-time -H:+ReportExceptionStackTraces -J-Xmx5g -H:-AddAllCharsets --enable-url-protocols=http,https -H:-UseServiceLoaderFeature -H:+StackTrace -J--add-exports=org.graalvm.sdk/org.graalvm.nativeimage.impl=ALL-UNNAMED -J--add-exports=org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk=ALL-UNNAMED -J--add-exports=org.graalvm.nativeimage.builder/com.oracle.svm.core.configure=ALL-UNNAMED -J--add-exports=org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk.localization=ALL-UNNAMED quarkus-integration-test-keycloak-authorization-999-SNAPSHOT-runner -jar quarkus-integration-test-keycloak-authorization-999-SNAPSHOT-runner.jar
2022-09-16T07:59:32.5291022Z WARNING: Unknown module: org.graalvm.nativeimage.llvm specified to --add-exports
2022-09-16T07:59:32.5295590Z WARNING: Unknown module: org.graalvm.nativeimage.llvm specified to --add-exports
2022-09-16T07:59:32.5300354Z WARNING: Unknown module: org.graalvm.nativeimage.llvm specified to --add-exports
2022-09-16T07:59:40.9447781Z ========================================================================================================================
2022-09-16T07:59:40.9454760Z GraalVM Native Image: Generating 'quarkus-integration-test-keycloak-authorization-999-SNAPSHOT-runner' (executable)...
2022-09-16T07:59:40.9455259Z ========================================================================================================================
2022-09-16T07:59:44.4842027Z [1/7] Initializing...                                                                                   (11.3s @ 0.34GB)
2022-09-16T07:59:44.4851510Z  Version info: 'GraalVM 22.2.0 Java 17 CE'
2022-09-16T07:59:44.4858487Z  Java version info: '17.0.4+8-jvmci-22.2-b06'
2022-09-16T07:59:44.4865343Z  C compiler: gcc (redhat, x86_64, 8.5.0)
2022-09-16T07:59:44.4871497Z  Garbage collector: Serial GC
2022-09-16T07:59:44.4885481Z  4 user-specific feature(s)
2022-09-16T07:59:44.4895194Z  - io.quarkus.runner.Feature: Auto-generated class by Quarkus from the existing extensions
2022-09-16T07:59:44.4904173Z  - io.quarkus.runtime.graal.DisableLoggingFeature: Disables INFO logging during the analysis phase for the [org.jboss.threads] categories
2022-09-16T07:59:44.4910003Z  - io.quarkus.runtime.graal.ResourcesFeature: Register each line in META-INF/quarkus-native-resources.txt as a resource on Substrate VM
2022-09-16T07:59:44.4915480Z  - org.graalvm.home.HomeFinderFeature: Finds GraalVM paths and its version number
2022-09-16T08:01:16.6072839Z [2/7] Performing analysis...  [*********]                                                               (91.9s @ 1.58GB)
2022-09-16T08:01:16.6103707Z   14,888 (91.28%) of 16,311 classes reachable
2022-09-16T08:01:16.6147908Z   27,540 (62.60%) of 43,994 fields reachable
2022-09-16T08:01:16.6432527Z   75,779 (58.56%) of 129,397 methods reachable
2022-09-16T08:01:16.6433170Z      735 classes, 1,178 fields, and 4,651 methods registered for reflection
2022-09-16T08:01:16.6434729Z       63 classes,    68 fields, and    55 methods registered for JNI access
2022-09-16T08:01:16.6435203Z        4 native libraries: dl, pthread, rt, z
2022-09-16T08:01:16.6456433Z 
2022-09-16T08:01:16.6623425Z Error: Class initialization of io.quarkus.runner.ApplicationImpl failed. Use the option --initialize-at-run-time=io.quarkus.runner.ApplicationImpl to explicitly request delayed initialization of this class.
2022-09-16T08:01:16.6623902Z Detailed message:
2022-09-16T08:01:16.6624029Z 
2022-09-16T08:01:16.6624680Z com.oracle.svm.core.util.UserError$UserException: Class initialization of io.quarkus.runner.ApplicationImpl failed. Use the option --initialize-at-run-time=io.quarkus.runner.ApplicationImpl to explicitly request delayed initialization of this class.
2022-09-16T08:01:16.6625259Z Detailed message:
2022-09-16T08:01:16.6625377Z 
2022-09-16T08:01:16.6625609Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.UserError.abort(UserError.java:84)
2022-09-16T08:01:16.6626144Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.FallbackFeature.reportAsFallback(FallbackFeature.java:249)
2022-09-16T08:01:16.6626757Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:762)
2022-09-16T08:01:16.6627349Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:564)
2022-09-16T08:01:16.6627892Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:521)
2022-09-16T08:01:16.6628528Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:407)
2022-09-16T08:01:16.6629349Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:585)
2022-09-16T08:01:16.6629958Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:128)
2022-09-16T08:01:16.6631372Z Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Class initialization of io.quarkus.runner.ApplicationImpl failed. Use the option --initialize-at-run-time=io.quarkus.runner.ApplicationImpl to explicitly request delayed initialization of this class.
2022-09-16T08:01:16.6631996Z Detailed message:
2022-09-16T08:01:16.6632119Z 
2022-09-16T08:01:16.6632445Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.constraints.UnsupportedFeatures.report(UnsupportedFeatures.java:126)
2022-09-16T08:01:16.6633094Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:759)
2022-09-16T08:01:16.6633488Z 	... 5 more
2022-09-16T08:01:16.6633855Z Caused by: java.lang.ExceptionInInitializerError
2022-09-16T08:01:16.6634233Z 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
2022-09-16T08:01:16.6634655Z 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1155)
2022-09-16T08:01:16.6635366Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ConfigurableClassInitialization.ensureClassInitialized(ConfigurableClassInitialization.java:182)
2022-09-16T08:01:16.6661717Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ConfigurableClassInitialization.computeInitKindAndMaybeInitializeClass(ConfigurableClassInitialization.java:625)
2022-09-16T08:01:16.6662831Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ConfigurableClassInitialization.computeInitKindAndMaybeInitializeClass(ConfigurableClassInitialization.java:135)
2022-09-16T08:01:16.6663836Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ConfigurableClassInitialization.maybeInitializeHosted(ConfigurableClassInitialization.java:173)
2022-09-16T08:01:16.6664541Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.initializeType(SVMHost.java:259)
2022-09-16T08:01:16.6665100Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.initializeType(AnalysisUniverse.java:670)
2022-09-16T08:01:16.6665682Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.lambda$new$0(AnalysisType.java:279)
2022-09-16T08:01:16.6666161Z 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
2022-09-16T08:01:16.6666556Z 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2022-09-16T08:01:16.6667032Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisFuture.ensureDone(AnalysisFuture.java:63)
2022-09-16T08:01:16.6667614Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.ensureInitialized(AnalysisType.java:628)
2022-09-16T08:01:16.6668197Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.onReachable(AnalysisType.java:552)
2022-09-16T08:01:16.6668771Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AtomicUtils.atomicMarkAndRun(AtomicUtils.java:52)
2022-09-16T08:01:16.6669360Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.lambda$registerAsReachable$8(AnalysisType.java:530)
2022-09-16T08:01:16.6669943Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:608)
2022-09-16T08:01:16.6670538Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:594)
2022-09-16T08:01:16.6671123Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:590)
2022-09-16T08:01:16.6671741Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.registerAsReachable(AnalysisType.java:530)
2022-09-16T08:01:16.6672613Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.ameta.AnalysisConstantReflectionProvider.registerAsReachable(AnalysisConstantReflectionProvider.java:334)
2022-09-16T08:01:16.6673342Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.ameta.HostedDynamicHubFeature.replace(HostedDynamicHubFeature.java:53)
2022-09-16T08:01:16.6673953Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.replaceObject(AnalysisUniverse.java:583)
2022-09-16T08:01:16.6674570Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.snippets.ReflectionPlugins.getIntrinsic(ReflectionPlugins.java:494)
2022-09-16T08:01:16.6675165Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.snippets.ReflectionPlugins.pushConstant(ReflectionPlugins.java:534)
2022-09-16T08:01:16.6675897Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.snippets.ReflectionPlugins.processClassForName(ReflectionPlugins.java:286)
2022-09-16T08:01:16.6676507Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.snippets.ReflectionPlugins$4.apply(ReflectionPlugins.java:218)
2022-09-16T08:01:16.6677101Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.graphbuilderconf.InvocationPlugin.execute(InvocationPlugin.java:329)
2022-09-16T08:01:16.6677709Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.applyInvocationPlugin(BytecodeParser.java:2252)
2022-09-16T08:01:16.6678405Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.AnalysisGraphBuilderPhase$AnalysisBytecodeParser.applyInvocationPlugin(AnalysisGraphBuilderPhase.java:89)
2022-09-16T08:01:16.6679084Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.tryInvocationPlugin(BytecodeParser.java:2191)
2022-09-16T08:01:16.6679757Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.AnalysisGraphBuilderPhase$AnalysisBytecodeParser.tryInvocationPlugin(AnalysisGraphBuilderPhase.java:70)
2022-09-16T08:01:16.6680402Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.appendInvoke(BytecodeParser.java:1875)
2022-09-16T08:01:16.6680947Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genInvokeStatic(BytecodeParser.java:1659)
2022-09-16T08:01:16.6681492Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genInvokeStatic(BytecodeParser.java:1639)
2022-09-16T08:01:16.6682022Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5224)
2022-09-16T08:01:16.6682588Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3359)
2022-09-16T08:01:16.6683175Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3319)
2022-09-16T08:01:16.6683732Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3164)
2022-09-16T08:01:16.6684245Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1138)
2022-09-16T08:01:16.6684752Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
2022-09-16T08:01:16.6685280Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:84)
2022-09-16T08:01:16.6685845Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:79)
2022-09-16T08:01:16.6686355Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
2022-09-16T08:01:16.6686798Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:261)
2022-09-16T08:01:16.6687240Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
2022-09-16T08:01:16.6687663Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
2022-09-16T08:01:16.6688249Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:135)
2022-09-16T08:01:16.6688905Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:685)
2022-09-16T08:01:16.6689626Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.lookupEncodedGraph(InlineBeforeAnalysis.java:180)
2022-09-16T08:01:16.6690316Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.doInline(PEGraphDecoder.java:1162)
2022-09-16T08:01:16.6690882Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.tryInline(PEGraphDecoder.java:1145)
2022-09-16T08:01:16.6691478Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.trySimplifyInvoke(PEGraphDecoder.java:1003)
2022-09-16T08:01:16.6692119Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.handleInvoke(PEGraphDecoder.java:957)
2022-09-16T08:01:16.6692686Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.processNextNode(GraphDecoder.java:817)
2022-09-16T08:01:16.6693330Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.processNextNode(InlineBeforeAnalysis.java:240)
2022-09-16T08:01:16.6693963Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.decode(GraphDecoder.java:548)
2022-09-16T08:01:16.6694478Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.decode(PEGraphDecoder.java:833)
2022-09-16T08:01:16.6695079Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysis.decodeGraph(InlineBeforeAnalysis.java:98)
2022-09-16T08:01:16.6695714Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:176)
2022-09-16T08:01:16.6696327Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:343)
2022-09-16T08:01:16.6696942Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:93)
2022-09-16T08:01:16.6697589Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:83)
2022-09-16T08:01:16.6698255Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraph(MethodTypeFlow.java:65)
2022-09-16T08:01:16.6698971Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultStaticInvokeTypeFlow.update(DefaultStaticInvokeTypeFlow.java:64)
2022-09-16T08:01:16.6699597Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:635)
2022-09-16T08:01:16.6700197Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
2022-09-16T08:01:16.6700970Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
2022-09-16T08:01:16.6755502Z 	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
2022-09-16T08:01:16.6755952Z 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
2022-09-16T08:01:16.6756425Z 	at java.base/java.util.concurrent.ForkJoinPool.externalHelpQuiescePool(ForkJoinPool.java:2104)
2022-09-16T08:01:16.6756903Z 	at java.base/java.util.concurrent.ForkJoinPool.awaitQuiescence(ForkJoinPool.java:3321)
2022-09-16T08:01:16.6757443Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.complete(CompletionExecutor.java:243)
2022-09-16T08:01:16.6758038Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis.doTypeflow(PointsToAnalysis.java:692)
2022-09-16T08:01:16.6758612Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis.finish(PointsToAnalysis.java:680)
2022-09-16T08:01:16.6759317Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis.runAnalysis(PointsToAnalysis.java:736)
2022-09-16T08:01:16.6759928Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:731)
2022-09-16T08:01:16.6760334Z 	... 5 more
2022-09-16T08:01:16.6760584Z Caused by: java.lang.RuntimeException: Failed to start quarkus
2022-09-16T08:01:16.6760994Z 	at io.quarkus.runner.ApplicationImpl.<clinit>(Unknown Source)
2022-09-16T08:01:16.6761244Z 	... 82 more
2022-09-16T08:01:16.6761566Z Caused by: java.lang.RuntimeException: java.lang.ClassNotFoundException: java.util.Map<?, ?>
2022-09-16T08:01:16.6762160Z 	at io.quarkus.resteasy.reactive.common.runtime.ResteasyReactiveCommonRecorder.loadClass(ResteasyReactiveCommonRecorder.java:53)
2022-09-16T08:01:16.6763010Z 	at io.quarkus.resteasy.reactive.common.runtime.ResteasyReactiveCommonRecorder.registerReader(ResteasyReactiveCommonRecorder.java:43)
2022-09-16T08:01:16.6763678Z 	at io.quarkus.deployment.steps.ResteasyReactiveProcessor$serverSerializers168685733.deploy_0(Unknown Source)
2022-09-16T08:01:16.6764339Z 	at io.quarkus.deployment.steps.ResteasyReactiveProcessor$serverSerializers168685733.deploy(Unknown Source)
2022-09-16T08:01:16.6764669Z 	... 83 more
2022-09-16T08:01:16.6764937Z Caused by: java.lang.ClassNotFoundException: java.util.Map<?, ?>
2022-09-16T08:01:16.6765353Z 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
2022-09-16T08:01:16.6766584Z 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
2022-09-16T08:01:16.6766949Z 	at java.base/jdk.internal.loader.Loader.loadClass(Loader.java:564)
2022-09-16T08:01:16.6767303Z 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
2022-09-16T08:01:16.6767621Z 	at java.base/java.lang.Class.forName0(Native Method)
2022-09-16T08:01:16.6767921Z 	at java.base/java.lang.Class.forName(Class.java:467)
2022-09-16T08:01:16.6768461Z 	at io.quarkus.resteasy.reactive.common.runtime.ResteasyReactiveCommonRecorder.loadClass(ResteasyReactiveCommonRecorder.java:51)
2022-09-16T08:01:16.6768913Z 	... 86 more
2022-09-16T08:01:16.6769447Z ------------------------------------------------------------------------------------------------------------------------
2022-09-16T08:01:16.6769855Z                         9.8s (9.4% of total time) in 34 GCs | Peak RSS: 3.24GB | CPU load: 1.95
2022-09-16T08:01:16.6770157Z ========================================================================================================================
2022-09-16T08:01:16.6770717Z Failed generating 'quarkus-integration-test-keycloak-authorization-999-SNAPSHOT-runner' after 1m 43s.
2022-09-16T08:01:16.7193459Z Error: Image build request failed with exit status 1
2022-09-16T08:01:16.8623070Z [INFO] ------------------------------------------------------------------------
2022-09-16T08:01:16.8623637Z [INFO] Reactor Summary for Quarkus - Integration Tests - Security WebAuthn 999-SNAPSHOT:
2022-09-16T08:01:16.8623926Z [INFO] 
2022-09-16T08:01:16.8624272Z [INFO] Quarkus - Integration Tests - Security WebAuthn .... SUCCESS [04:55 min]
2022-09-16T08:01:16.8624722Z [INFO] Quarkus - Integration Tests - Keycloak Policy Enforcer FAILURE [01:54 min]
2022-09-16T08:01:16.8625165Z [INFO] ------------------------------------------------------------------------
2022-09-16T08:01:16.8625577Z [INFO] BUILD FAILURE
2022-09-16T08:01:16.8625983Z [INFO] ------------------------------------------------------------------------
2022-09-16T08:01:16.8626267Z [INFO] Total time:  06:53 min
2022-09-16T08:01:16.8626562Z [INFO] Finished at: 2022-09-16T08:01:16Z
2022-09-16T08:01:16.8627089Z [INFO] ------------------------------------------------------------------------
2022-09-16T08:01:16.9076480Z [ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:build (default) on project quarkus-integration-test-keycloak-authorization: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
2022-09-16T08:01:16.9077862Z [ERROR] 	[error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed. Exit code: 1
2022-09-16T08:01:16.9094049Z [ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:417)
2022-09-16T08:01:16.9097467Z [ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:258)
2022-09-16T08:01:16.9100692Z [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2022-09-16T08:01:16.9103989Z [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9106989Z [ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9110105Z [ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2022-09-16T08:01:16.9113183Z [ERROR] 	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
2022-09-16T08:01:16.9116002Z [ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
2022-09-16T08:01:16.9118777Z [ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
2022-09-16T08:01:16.9121588Z [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
2022-09-16T08:01:16.9124443Z [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
2022-09-16T08:01:16.9127207Z [ERROR] 	at java.base/java.lang.Thread.run(Thread.java:829)
2022-09-16T08:01:16.9130259Z [ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
2022-09-16T08:01:16.9133506Z [ERROR] -> [Help 1]
2022-09-16T08:01:16.9137835Z org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:build (default) on project quarkus-integration-test-keycloak-authorization: Failed to build quarkus application
2022-09-16T08:01:16.9142874Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:375)
2022-09-16T08:01:16.9146859Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
2022-09-16T08:01:16.9150356Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
2022-09-16T08:01:16.9153757Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
2022-09-16T08:01:16.9157122Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
2022-09-16T08:01:16.9160578Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
2022-09-16T08:01:16.9164368Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
2022-09-16T08:01:16.9168172Z     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
2022-09-16T08:01:16.9171832Z     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
2022-09-16T08:01:16.9175198Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
2022-09-16T08:01:16.9178495Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
2022-09-16T08:01:16.9182194Z     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
2022-09-16T08:01:16.9185485Z     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
2022-09-16T08:01:16.9188752Z     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
2022-09-16T08:01:16.9192002Z     at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
2022-09-16T08:01:16.9195287Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2022-09-16T08:01:16.9199186Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9202856Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9206156Z     at java.lang.reflect.Method.invoke (Method.java:566)
2022-09-16T08:01:16.9208801Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2022-09-16T08:01:16.9211052Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2022-09-16T08:01:16.9213672Z     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2022-09-16T08:01:16.9215932Z     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2022-09-16T08:01:16.9218223Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2022-09-16T08:01:16.9220694Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9223613Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9225884Z     at java.lang.reflect.Method.invoke (Method.java:566)
2022-09-16T08:01:16.9227898Z     at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:53)
2022-09-16T08:01:16.9230279Z     at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:152)
2022-09-16T08:01:16.9232653Z     at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:76)
2022-09-16T08:01:16.9235081Z Caused by: org.apache.maven.plugin.MojoExecutionException: Failed to build quarkus application
2022-09-16T08:01:16.9237859Z     at io.quarkus.maven.BuildMojo.doExecute (BuildMojo.java:165)
2022-09-16T08:01:16.9240369Z     at io.quarkus.maven.QuarkusBootstrapMojo.execute (QuarkusBootstrapMojo.java:154)
2022-09-16T08:01:16.9242876Z     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
2022-09-16T08:01:16.9245321Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
2022-09-16T08:01:16.9247626Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
2022-09-16T08:01:16.9250008Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
2022-09-16T08:01:16.9252350Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
2022-09-16T08:01:16.9254956Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
2022-09-16T08:01:16.9257378Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
2022-09-16T08:01:16.9259747Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
2022-09-16T08:01:16.9262260Z     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
2022-09-16T08:01:16.9264819Z     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
2022-09-16T08:01:16.9267071Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
2022-09-16T08:01:16.9269273Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
2022-09-16T08:01:16.9271764Z     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
2022-09-16T08:01:16.9274191Z     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
2022-09-16T08:01:16.9276221Z     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
2022-09-16T08:01:16.9277816Z     at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
2022-09-16T08:01:16.9279696Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2022-09-16T08:01:16.9281507Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9283979Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9286267Z     at java.lang.reflect.Method.invoke (Method.java:566)
2022-09-16T08:01:16.9288017Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2022-09-16T08:01:16.9289767Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2022-09-16T08:01:16.9291519Z     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2022-09-16T08:01:16.9294283Z     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2022-09-16T08:01:16.9295872Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2022-09-16T08:01:16.9297609Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9299436Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9301312Z     at java.lang.reflect.Method.invoke (Method.java:566)
2022-09-16T08:01:16.9303353Z     at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:53)
2022-09-16T08:01:16.9305141Z     at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:152)
2022-09-16T08:01:16.9306865Z     at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:76)
2022-09-16T08:01:16.9308437Z Caused by: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
2022-09-16T08:01:16.9309141Z 	[error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed. Exit code: 1
2022-09-16T08:01:16.9309891Z 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:417)
2022-09-16T08:01:16.9310476Z 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:258)
2022-09-16T08:01:16.9310961Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2022-09-16T08:01:16.9311457Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9312005Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9312444Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2022-09-16T08:01:16.9312822Z 	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
2022-09-16T08:01:16.9313201Z 	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
2022-09-16T08:01:16.9313564Z 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
2022-09-16T08:01:16.9313962Z 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
2022-09-16T08:01:16.9314408Z 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
2022-09-16T08:01:16.9314779Z 	at java.base/java.lang.Thread.run(Thread.java:829)
2022-09-16T08:01:16.9315092Z 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
2022-09-16T08:01:16.9316280Z 
2022-09-16T08:01:16.9319515Z     at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment (AugmentActionImpl.java:332)
2022-09-16T08:01:16.9321417Z     at io.quarkus.runner.bootstrap.AugmentActionImpl.createProductionApplication (AugmentActionImpl.java:175)
2022-09-16T08:01:16.9323203Z     at io.quarkus.maven.BuildMojo.doExecute (BuildMojo.java:132)
2022-09-16T08:01:16.9324859Z     at io.quarkus.maven.QuarkusBootstrapMojo.execute (QuarkusBootstrapMojo.java:154)
2022-09-16T08:01:16.9327148Z     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
2022-09-16T08:01:16.9328784Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
2022-09-16T08:01:16.9356332Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
2022-09-16T08:01:16.9357003Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
2022-09-16T08:01:16.9357489Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
2022-09-16T08:01:16.9357944Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
2022-09-16T08:01:16.9358963Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
2022-09-16T08:01:16.9359597Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
2022-09-16T08:01:16.9360277Z     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
2022-09-16T08:01:16.9360891Z     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
2022-09-16T08:01:16.9361337Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
2022-09-16T08:01:16.9361827Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
2022-09-16T08:01:16.9362208Z     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
2022-09-16T08:01:16.9362560Z     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
2022-09-16T08:01:16.9362907Z     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
2022-09-16T08:01:16.9363252Z     at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
2022-09-16T08:01:16.9363629Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2022-09-16T08:01:16.9364101Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9364631Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9365064Z     at java.lang.reflect.Method.invoke (Method.java:566)
2022-09-16T08:01:16.9365481Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2022-09-16T08:01:16.9365974Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2022-09-16T08:01:16.9366466Z     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2022-09-16T08:01:16.9366939Z     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2022-09-16T08:01:16.9367377Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2022-09-16T08:01:16.9367843Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9368362Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9368781Z     at java.lang.reflect.Method.invoke (Method.java:566)
2022-09-16T08:01:16.9369177Z     at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:53)
2022-09-16T08:01:16.9369635Z     at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:152)
2022-09-16T08:01:16.9370087Z     at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:76)
2022-09-16T08:01:16.9370502Z Caused by: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
2022-09-16T08:01:16.9371172Z 	[error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed. Exit code: 1
2022-09-16T08:01:16.9371927Z 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:417)
2022-09-16T08:01:16.9372503Z 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:258)
2022-09-16T08:01:16.9372978Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2022-09-16T08:01:16.9373470Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9374016Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9374510Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2022-09-16T08:01:16.9374881Z 	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
2022-09-16T08:01:16.9375261Z 	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
2022-09-16T08:01:16.9375624Z 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
2022-09-16T08:01:16.9376020Z 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
2022-09-16T08:01:16.9376462Z 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
2022-09-16T08:01:16.9376829Z 	at java.base/java.lang.Thread.run(Thread.java:829)
2022-09-16T08:01:16.9377138Z 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
2022-09-16T08:01:16.9377326Z 
2022-09-16T08:01:16.9377482Z     at io.quarkus.builder.Execution.run (Execution.java:123)
2022-09-16T08:01:16.9377932Z     at io.quarkus.builder.BuildExecutionBuilder.execute (BuildExecutionBuilder.java:79)
2022-09-16T08:01:16.9378370Z     at io.quarkus.deployment.QuarkusAugmentor.run (QuarkusAugmentor.java:158)
2022-09-16T08:01:16.9378816Z     at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment (AugmentActionImpl.java:328)
2022-09-16T08:01:16.9379378Z     at io.quarkus.runner.bootstrap.AugmentActionImpl.createProductionApplication (AugmentActionImpl.java:175)
2022-09-16T08:01:16.9379860Z     at io.quarkus.maven.BuildMojo.doExecute (BuildMojo.java:132)
2022-09-16T08:01:16.9380252Z     at io.quarkus.maven.QuarkusBootstrapMojo.execute (QuarkusBootstrapMojo.java:154)
2022-09-16T08:01:16.9380889Z     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
2022-09-16T08:01:16.9381428Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
2022-09-16T08:01:16.9381921Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
2022-09-16T08:01:16.9382390Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
2022-09-16T08:01:16.9382859Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
2022-09-16T08:01:16.9383312Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
2022-09-16T08:01:16.9383869Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
2022-09-16T08:01:16.9384470Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
2022-09-16T08:01:16.9385150Z     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
2022-09-16T08:01:16.9385783Z     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
2022-09-16T08:01:16.9386230Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
2022-09-16T08:01:16.9386604Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
2022-09-16T08:01:16.9386978Z     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
2022-09-16T08:01:16.9387339Z     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
2022-09-16T08:01:16.9387685Z     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
2022-09-16T08:01:16.9388027Z     at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
2022-09-16T08:01:16.9388422Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2022-09-16T08:01:16.9388894Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9389421Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9389849Z     at java.lang.reflect.Method.invoke (Method.java:566)
2022-09-16T08:01:16.9390272Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2022-09-16T08:01:16.9390807Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2022-09-16T08:01:16.9391301Z     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2022-09-16T08:01:16.9391784Z     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2022-09-16T08:01:16.9392227Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2022-09-16T08:01:16.9392679Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9393203Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9393627Z     at java.lang.reflect.Method.invoke (Method.java:566)
2022-09-16T08:01:16.9394016Z     at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:53)
2022-09-16T08:01:16.9394526Z     at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:152)
2022-09-16T08:01:16.9394971Z     at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:76)
2022-09-16T08:01:16.9395503Z Caused by: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed. Exit code: 1
2022-09-16T08:01:16.9396130Z     at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed (NativeImageBuildStep.java:417)
2022-09-16T08:01:16.9396703Z     at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build (NativeImageBuildStep.java:258)
2022-09-16T08:01:16.9397171Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2022-09-16T08:01:16.9397633Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2022-09-16T08:01:16.9398148Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2022-09-16T08:01:16.9398571Z     at java.lang.reflect.Method.invoke (Method.java:566)
2022-09-16T08:01:16.9398939Z     at io.quarkus.deployment.ExtensionLoader$3.execute (ExtensionLoader.java:909)
2022-09-16T08:01:16.9399307Z     at io.quarkus.builder.BuildContext.run (BuildContext.java:281)
2022-09-16T08:01:16.9399674Z     at org.jboss.threads.ContextHandler$1.runWith (ContextHandler.java:18)
2022-09-16T08:01:16.9400083Z     at org.jboss.threads.EnhancedQueueExecutor$Task.run (EnhancedQueueExecutor.java:2449)
2022-09-16T08:01:16.9400533Z     at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run (EnhancedQueueExecutor.java:1478)
2022-09-16T08:01:16.9400882Z     at java.lang.Thread.run (Thread.java:829)
2022-09-16T08:01:16.9401189Z     at org.jboss.threads.JBossThread.run (JBossThread.java:501)
2022-09-16T08:01:16.9401443Z [ERROR] 
```

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

This message is reported:
```
Caused by: java.lang.RuntimeException: java.lang.ClassNotFoundException: java.util.Map<?, ?>
2022-09-16T08:01:16.6762160Z 	at io.quarkus.resteasy.reactive.common.runtime.ResteasyReactiveCommonRecorder.loadClass(ResteasyReactiveCommonRecorder.java:53)
2022-09-16T08:01:16.6763010Z 	at io.quarkus.resteasy.reactive.common.runtime.ResteasyReactiveCommonRecorder.registerReader(ResteasyReactiveCommonRecorder.java:43)
2022-09-16T08:01:16.6763678Z 	at io.quarkus.deployment.steps.ResteasyReactiveProcessor$serverSerializers168685733.deploy_0(Unknown Source)
2022-09-16T08:01:16.6764339Z 	at io.quarkus.deployment.steps.ResteasyReactiveProcessor$serverSerializers168685733.deploy(Unknown Source)
2022-09-16T08:01:16.6764669Z 	... 83 more
```

--- a/integration-tests/keycloak-authorization/pom.xml
+++ b/integration-tests/keycloak-authorization/pom.xml
@@ -51,24 +51,6 @@
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-admin-client</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>jakarta.activation</groupId>
-                    <artifactId>jakarta.activation-api</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.jboss.spec.javax.ws.rs</groupId>
-                    <artifactId>jboss-jaxrs-api_3.0_spec</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>com.sun.activation</groupId>
-                    <artifactId>jakarta.activation</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
 
         <!-- Minimal test dependencies to *-deployment artifacts for consistent build order -->
         <dependency>

